types/twine-sugarcube/macro.d.ts

```diff
@@ -119,8 +142,20 @@ export interface MacroContext {
 
 export interface MacroDefinition {
     handler: (this: MacroContext) => void;
-    tags?: string[] | null | undefined;
-    skipArgs?: boolean | undefined;
+    /**
+     * Having this property signifies that this is a container macro
+     * This should be an array of child tag names or `null`
+     * @since 2.0.0
+     */
+    tags?: string[] | null;
```

Comment:
Please revert the `undefined`-related changes (see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54250).

Reply:
Alright, a new DT or TS thing.  Seems stupid to me, but whatever. 🙄 

Anyway.  I stand corrected.

Reply:
Done.